docs/quickstart.md

```diff
@@ -0,0 +1,293 @@
+
+Combining `pynapple` and `nemos` can greatly streamline the modeling process. To show how handy this could 
+be, let's simulate a common scenario for many systems neuro-scientists: we record a behavioral
+feature with a temporal resolution, but we want to model spike counts at a different resolution. On top of that,
+let's assume that we want to restrict our fit to a specific time epoch.
+
+All of this could be done in a single line of code.
+
+First, let's simulate our feature and the spike times of a neuron, and define the recording epoch and the 
+epoch that we want to use for fitting (let's assume it marks when the subject was awake).
+
+```python
+import numpy as np
+import pynapple as nap
+
+# Assume that this is your input before any processing:
+# - recording duration: 5 min 
+# - task variable sampled at 10KHz
+# - spike times in seconds.
+
+# define time axis and a feature
+time_sec = np.linspace(0, 300, 300 * 10000)
+X = nap.TsdTensor(t=time_sec, d=0.2 * np.random.normal(size=(time_sec.shape[0], 1, 1)))
+
+# define epochs as pynapple IntervalSet
+recording_epoch = nap.IntervalSet(start=0, end=300)
+wake_epoch = nap.IntervalSet(start=0, end=150)           
+
+# deine spikes by generating random times, and adding the recording_epoch as the time_support for the spikes
+spike_ts = nap.Ts(
+    np.sort(np.random.uniform(0, 300, size=100)), 
+    time_support=recording_epoch
+)
+spikes = nap.TsGroup({1: spike_ts})
+```
+
+Finally, let's process and fit our data.
+
+```python
+import nemos as nmo
+
+# Actual processing and fitting:
+# - Restrict to the wake epoch
+# - Down-sample to 10ms the features
+# - Bin the spike to the same resolution
+# - Fit a GLM.
+nmo.glm.GLM().fit(X.bin_average(0.01).restrict(wake_epoch), spikes.count(0.01).restrict(wake_epoch))
+```
+
+!!! note
+    In this example we do all the processing and fitting in a single line to showcase how versatile this approach can
+    be. In general, you should always avoid nesting many processing steps without each transformation: what if you
+    unintentionally used the wrong bin-size? What if you selected the wrong feature? 
+
+### Interactions with `scikit-learn`
+
+As previously mentioned, `nemos` GLM conforms to the `scikit-learn` API for estimators. As a consequence, 
+you can retrieve the all the parameters and set any of them using the `get_param` and `set_param` methods.
+
+```python
+import nemos as nmo
+
+model = nmo.glm.GLM()
+
+# get a dictionary of attributes, including nested once.
+parameter_dict = model.get_params()
+print(parameter_dict)
+
+
+# set a model attribute
+model.set_params(regularizer=nmo.regularizer.Lasso())
+print(model.get_params()["regularizer"])
+
+# set a nested attribute
+model.set_params(regularizer__regularizer_strength=10)
+print(model.get_params())
+```
+
+This is used internally by `scikit-learn` to construct complex pipelines.
+
+#### Why should you care?
```

Comment:
I think this sub-section should be the only scikit learn material presented in the quickstart

Reply:
I reduced the previous session to a single sentence